libraries/rib-base/src/main/java/com/badoo/ribs/core/view/RibView.kt

```diff
@@ -41,5 +42,9 @@ interface RibView {
      * Detaches a child from this view. Acts as the opposite of [attachChild].
      */
     fun detachChild(child: Node<*>, subtreeOf: Node<*>)
+
+    interface LifecycleAware {
```

Comment:
This one doesn't extend Plugin?

Reply:
This one should not be confused with ViewAware, which is actual plugin.
This one is more like extension for view implementation.